e2e/runner/run_cypress_ci.js

```diff
@@ -5,7 +5,15 @@ const { printBold } = require("./cypress-runner-utils");
 
 const mode = process.argv?.[2]?.trim();
 
-const availableModes = ["start", "snapshot", "e2e", "component"];
+const availableModes = [
+  "start",
+  "snapshot",
+  "e2e",
+  "component",
+  "metabase-nodejs-react-sdk-embedding-sample-e2e",
+  "metabase-nextjs-sdk-embedding-sample-e2e",
+  "shoppy-e2e",
```

Comment:
This feels a bit strange as we already have the `"e2e"` mode above, and we're adding 3 new e2e modes - one for each sample app.

Reply:
The name of that constant is a bit wrong. We describe `test suites` here, so we have a default `e2e` test suite for `metabase`, `components` test suite, and a test suite for each sample app

Reply:
Let me rename it

Reply:
ah, I saw the `start` and `snapshot` mode so I thought it's like a group of operations that `run_cypress_ci` can do

Reply:
yeah, tbh now i even don't know what naming is better.
`modeOrTestSuite`?

Reply:
let me split it into 2 arrays

Reply:
Added a commit

Reply:
Thanks!